source/form.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Подбор программы</title>
+  </head>
+  <body class="page__body" style="width: 320px; height: 2935px;">
+    <header class="main-header">
+      <!--ЛОГО-->
+      <a class="main-header__logo" href="#">
+        <img
+          src="img/logo-mobile.svg"
+          width="33" height="38"
+          alt="логотип магазина.">
+      </a>
+      <a class="main-header__logo-text" href="#">
+        <img src="img/logo-text.svg"
+          width="101" height="18"
+          alt="Кэт энерджи.">
+      </a>
+      <!--НАВИГАЦИЯ-->
+      <nav class="main-nav">
+        <!--<button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>-->
+        <input class="burger-checkbox" type="checkbox" id="burger-checkbox">
+        <label class="burger" for="burger-checkbox"></label>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="list__item">
+              <a href="index.html">главная</a>
+            </li>
+            <li class="list__item">
+              <a href="catalog.html">каталог продукции</a>
+            </li>
+            <li class="list__item list__item--active"><a href="#">подбор
+                программы
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!--MAIN-->
+    <main class="main-container main__form">
+      <!--! Q не уверен в правильности именования класса секции-->
+      <section class="questionary">
+        <div class="questionary__wrapper">
+          <h2 class="questionary__title">Подбор программы</h2>
+          <p class="questionary__text">
+            <!--! Q не уверен в правильности именования класса-->
+            Заполните анкету, и мы подберем
+            <br>программу питания для вашего кота
+          </p>
+          <form
+            class="questionary__form"
+            action="https://echo.html.ru"
+            method="post"> <!--!название класса для form - масло масляное-->
+            <fieldset class="cat-details"> <!--!-->
+              <legend class="visually-hidden"><h3>Персональные данные
+                  кота</h3></legend>
+              <!--! Q или лучше не использовать legenda и оставить скрытый заголовок h3?-->
+              <label for="questionary__name">Имя<sup>*</sup></label>
```

Comment:
```suggestion
              <label for="questionary__name">Имя*</label>
```
Нельзя использовать теги просто так, они должны соответствовать семантике. Звездочка это не верхний индекс, даже если дизайнер её нарисовал чуть повыше основного текста. 
Если обертка нужна исключительно для стилизации, то span

Reply:
Не согласен. Звездочка смотрится как верхний индекс и похожий пример есть в демке Барбершопа
![1 sup](https://github.com/htmlacademy-adaptive/2426581-cat-energy-1/assets/148534470/cb54c6c8-ec5c-4dc6-b83e-b429c5ac35cb)